pages/utils/job_utils.py

```diff
@@ -36,42 +27,92 @@
     font=dict(size=18, color="orange"),
 )
 
+def _loading_graph(done, queued, retry, failed):
```

Comment:
Can you add some doc strings/comments in?

Reply:
yep added it to the revision list.

Reply:
@JamesKunstle sounds good, will mark as resolved once fixed

Reply:
added!